contents/handbook/company/management.md

```diff
@@ -15,12 +21,16 @@ A manager at PostHog has two tasks:
 That's it.
 
 A manager at PostHog is _not_ responsible for:
-1. Setting compensation (we have transparent compensation)
+1. Setting compensation (we have a [compensation calculator](/handbook/people/compensation) and increases are done occasionally by the people team).
 1. Setting tasks for their direct reports
 1. Creating a career path (career paths should be transparent and documented, and for now centrally managed)
 1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
 1. Giving feedback (managers give feedback in their capacity as individual contributor, but so does everyone else)
 
+## How do I make sure my direct reports are happy and productive?
+
+1. Do [regular 1:1s](/handbook/company/1-1s) every week or every other week
+1. Do a [performance review](/handbook/people/feedback#performance-reviews) every six months
```

Comment:
I think the numbered list isn't working at the moment

Reply:
Markdown let's you use `1.` for each point and will automatically render it as `1., 2.`, avoids you having to manually fix or re-write them numbers!